components/ActionSheet/docs/color-theming.md

```diff
@@ -1,39 +0,0 @@
-### Color Theming
```

Comment:
Please keep this original documentation until we've deleted the themers. Annotate these docs that they are representing a to-be-deleted set of APIs though so that their state is clear.

Reply:
Done